README.md

```diff
@@ -185,15 +185,15 @@ The test tool has the following limitations. If the DUT falls under these limita
 | R0124           | CreateContextStateWithAssociation                                                    |
 | R0125           | CreateContextStateWithAssociation                                                    |
 | R0133           | CreateContextStateWithAssociation                                                    |
-| B-61            | SetMetricQualityValidity                                                             |
```

Comment:
Den Test scheint es noch zu geben und er scheint auch noch die Manipulation SetMetricQualityValidity zu verwenden. Warum sollte der Eintrag in der Tabelle entfernt werden?

Reply:
You removed this test a few hours ago (see https://github.com/Draegerwerk/SDCcc/pull/77)

Reply:
Indeed. This change had not yet reached my main. I had to sync my fork to see it.